en/docs/install-and-setup/install/installing-the-product/installing-the-binary/install-mi-in-vm-binary.md

```diff
@@ -18,7 +17,7 @@ Extract the download ZIP file to a location on your computer.
 
 ## Prerequisites
 
-**Setting the Java_Home**: Set up a [JDK that is compatible with WSO2 Enterprise Integrator](../install_prerequisites/#environment-compatibility) and point the `java_home` variable to your JDK instance.
+**Setting the Java_Home**: Set up a [JDK that is compatible with the Micro Integrator](../install_prerequisites/#environment-compatibility) and point the `java_home` variable to your JDK instance.
```

Comment:
Shall we fix this link?

Reply:
Cannot fix this until the 'Install' section is fixed. This page currently does not exist.